src/stream_wrap.h

```diff
@@ -28,6 +28,10 @@
 #include "handle_wrap.h"
 #include "v8.h"
 
+#define STREAM_WRAP_RECVQ_INITIAL   16*1024
+#define STREAM_WRAP_RECVQ_MIN       16*1024
+#define STREAM_WRAP_RECVQ_MAX       256*1024
```

Comment:
```suggestion
constexpr ssize_t STREAM_WRAP_RECVQ_INITIAL = 16 * 1024;
constexpr ssize_t STREAM_WRAP_RECVQ_MIN = 16 * 1024;
constexpr ssize_t STREAM_WRAP_RECVQ_MAX = 256 * 1024;
```
Should we use `constexpr` here?